apps/air-discount-scheme/web/lib/environment.ts

```diff
@@ -0,0 +1,30 @@
+import { getStaticEnv } from '@island.is/shared/utils'
+
+const isProd = process.env.NODE_ENV === 'production'
+// TODO remove isLocalhost flag before release.
+const isLocalhost = false
+
+const devConfig = {
+  production: false,
+  NEXTAUTH_URL: isLocalhost ? 'http://localhost:4200' : process.env.NEXTAUTH_URL,
+  IDS: process.env.IDENTITY_SERVER_ISSUER_URL ?? 'identity-server.dev01.devland.is',
+  identityServerDomain:
+    process.env.IDENTITY_SERVER_DOMAIN ?? 'identity-server.dev01.devland.is',
+  identityServerLogoutURL: isLocalhost
+    ? 'https://loftbru.dev01.devland.is'
+    : getStaticEnv('NEXTAUTH_URL'),
+  identityServerSecret: process.env.IDENTITY_SERVER_SECRET,
+  idsCookieName: process.env.IDS_COOKIE_NAME ?? 'next-auth.session-token',
+}
+
+const prodConfig = {
+  production: true,
+  NEXTAUTH_URL: isProd ? getStaticEnv('NEXTAUTH_URL') : '',
+  IDS: isProd ? getStaticEnv('SI_PUBLIC_IDENTITY_SERVER_ISSUER_URL') : '',
+  identityServerDomain: isProd ? getStaticEnv('IDENTITY_SERVER_DOMAIN') : '',
+  identityServerLogoutURL: isProd ? getStaticEnv('NEXTAUTH_URL') : '',
+  identityServerSecret: process.env.IDENTITY_SERVER_SECRET,
+  idsCookieName: isProd ? getStaticEnv('IDS_COOKIE_NAME') : '',
```

Comment:
https://docs.devland.is/#environment-variables-with-static-websites
Should `getStaticEnv` be used for devConfig? (see notes)

Also, shouldn't the prefixes be consistent with `SI_PUBLIC_` as per the documentation and implementation: `getStaticEnv` has
```javascript
export const ENV_PREFIX = 'SI_PUBLIC_'

// then later
if (!environmentVariableName.startsWith(ENV_PREFIX)) {
  throw new Error(`Variable must be prefixed with ${ENV_PREFIX}`)
}
```

Reply:
@kksteini fixed.